src/device-registry/routes/v1/devices.js

```diff
@@ -1112,21 +1106,15 @@ router.post(
       )
       .bail()
       .notEmpty()
-      .withMessage("the name should not be empty if provided")
-      .bail()
-      .matches(/^[a-zA-Z0-9]+$/)
```

Comment:
no need to do input validations for `long_name`  and `name` during soft device creation